site/_posts/2021-10-26-how-to-set-up-single-sign-on-between-fusionauth-wordpress.md

```diff
@@ -0,0 +1,199 @@
+---
+layout: blog-post
+title: How to Set Up Single Sign-On Between FusionAuth and WordPress
+description: In this tutorial, you'll be learning how to implement OAuth SSO for Joomla users.
+author: Aniket Bhattacharyea
+image: blogs/wordpress-sso-fusionauth/how-to-set-up-single-sign-on-sso-between-fusionauth-and-wordpress-header-image.png
+category: blog
+tags: tutorial tutorial-php client-php tutorial-integration
+excerpt_separator: "<!--more-->"
+---
+
+User authentication is one of the most important parts of building any application, and almost every application features some sort of authentication. However, it quickly gets difficult to manage when you have multiple related but independent applications to maintain.
+
+<!--more-->
+
+Let's say you're building a photo-sharing application that lets a user sign up with a username and password to share their pictures. Your team might later decide to [create a forum using WordPress](https://wordpress.com/go/tutorials/create-a-forum-with-wordpress-com/) to help support your app, but this new project isn't tied to your original application. It uses an entirely different tech stack, database, and hosting setup.
```

Comment:
```suggestion
Let's say you're building a photo-sharing application that lets a user sign up with a username and password to share their pictures. Your team might later decide to [create a forum using WordPress](https://wordpress.com/go/tutorials/create-a-forum-with-wordpress-com/) so that your user's friends and family can comment on the photos, but this new project isn't tied to your original application. It uses an entirely different tech stack, database, and hosting setup.
```

Reply:
Tweaked in a different way.